cmd/oapi-codegen/oapi-codegen.go

```diff
@@ -113,8 +113,13 @@ func main() {
 		os.Exit(1)
 	}
 
+	// fallback to the old config style if we don't receive the new `--config` flag
```

Comment:
This won't work, because `--config` has been around for a long time, with a different format config file.